assets/components/LogViewer/ComplexLogItem.vue

```diff
@@ -39,25 +51,18 @@ const [expanded, expandToggle] = useToggle();
 const validValues = computed(() => {
   return Object.fromEntries(Object.entries(logEntry.message).filter(([_, value]) => value !== undefined));
 });
+
+const isHovering = ref(false);
 </script>
 
 <style lang="postcss" scoped>
 .text-light {
   @apply text-base-content/70;
 }
-.fields {
-  &:hover {
```

Comment:
This is how I was doing it before. When possible, _always_ prefer CSS over javascript.